hail_search/queries/mito.py

```diff
@@ -331,7 +331,7 @@ def _add_project_lookup_data(self, ht, annotation_fields, *args, **kwargs):
 
         annotation_fields.update({
             'familyGenotypes': lambda r: hl.dict(r.family_entries.map(
-                lambda entries: (entries.first().familyGuid, entries.map(self._get_sample_genotype))
+                lambda entries: (entries.first().familyGuid, entries.filter(hl.is_defined).map(self._get_sample_genotype))
```

Comment:
Why would a family entry be None here?

Reply:
also is it worth adding a test case for this?

Reply:
If there was no call for one sample at a specific site, it would end up as missing in the entries table

And probably. We do not have a table with. missing entry but its probably worth adding